.tekton/tasks/task-switchboard.yaml

```
@@ -51,13 +51,28 @@ spec:
 
         pr_number=$(gh search prs --repo konflux-ci/build-definitions "$(params.revision)" --json number --jq '.[].number')
 
-        ec opa eval --v1-compatible --data "${rules}" --input \
-        <(gh pr view "https://github.com/konflux-ci/build-definitions/pull/${pr_number}" --json files --jq '[.files.[].path']) \
-        'data[_]' \
-        | jq '[.result.[].expressions.[].value | to_entries | .[] | select(.value == true) | .key]' \
-        | tee "$(results.bindings.path)"
+        if [[ -z "${pr_number}" ]]; then
+          # Fetch the main branch for comparison
+          git fetch origin main
+          changed_files=$(git diff --name-only "origin/main..HEAD" | tr "\n" " ")
```

Situation: my PR doesn't update any files that should trigger e2e-tests, but it is outdated compared to main. In main, some files that trigger e2e-tests have been updated. With this approach, we would trigger e2e-tests because the relevant files changed in main.

I think we need the three-dot diff
```suggestion
          changed_files=$(git diff --name-only "origin/main...HEAD" | tr "\n" " ")
```
```
       git diff [<options>] <commit>...<commit> [--] [<path>...]
           This form is to view the changes on the branch containing and up to the second <commit>, starting at a common ancestor of both <commit>.  git diff A...B is
           equivalent to git diff $(git merge-base A B) B. You can omit any one of <commit>, which has the same effect as using HEAD instead.
```